library/SimplePie.php

```diff
@@ -3146,6 +3146,12 @@ public static function sort_items($a, $b)
 	{
 		$a_date = $a->get_date('U');
 		$b_date = $b->get_date('U');
+		if(version_compare(phpversion(), '7.0.0', '>=')) {
```

Comment:
Remove the `version_compare()`. The code you have for PHP 7 is actually right code for all platforms.

Reply:
This block should be existed because it separates the PHP 5.6 and PHP 7.
And it also prevents the unexpected ```usort``` behavior.

Reply:
why do you want to separate 5.6 & 7?
what is the unexpected `usort` behaviour?

Reply:
yes so it's the test using identical timestamps that expects an unreasonable value that's causing the problem here. I would just update tests/oldtests/first_item_title/SPtests/bugs/179.0.10.php

Reply:
Hi @peter279k I was looking at merging your PR but noticed this change hasn't been reverted yet. Can you do that first please?

Reply:
Thank you for your reply, @mblaney.
I have removed this.

Thanks.